spec/factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    sequence :email do |n|
+      "dummyEmail#{n}@gmail.com"
```

Comment:
We should change this to `@example.com` because email addresses at Gmail might be real, and we don't want to unintentionally spam them.  example.com is a special domain name that was specifically created for this purpose (for testing things like emails).